src/plugins/default/template/mup.js.sample

```diff
@@ -3,7 +3,7 @@ module.exports = {
     one: {
       // TODO: set host address, username, and authentication method
       host: '1.2.3.4',
-      username: 'root',
```

Comment:
Please leave the hanging commas in this file. That way users won't need to remember to add commas when they uncomment or add properties.